tests/service-catalog/Dockerfile

```diff
@@ -15,11 +15,13 @@ RUN CGO_ENABLED=0 go test -tags=acceptance -c ./test -o /servicecatalog.test
 FROM alpine:3.10
 RUN apk add --no-cache curl
 
+WORKDIR /app
```

Comment:
this influences the following `COPY` and `ENTRYPOINT` arguments, please make sure they are unified about the workdir.